.github/workflows/deploy.yml

```diff
@@ -163,6 +163,137 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
+  android_hybrid:
+    name: Build and deploy Android HybridApp
+    needs: prep
+    runs-on: ubuntu-latest-xl
```

Comment:
It seems like we currently run the workflow on `macos-13-xlarge`, do we want to change that?

Reply:
I think we want to build Android on ubuntu and [double checked thats what it's running on now](https://github.com/Expensify/Mobile-Deploy/blob/main/.github/workflows/deploy.yml#L282)